Notifiable-iOS/Category/NSUserDefaults+FWTNotifiable.m

```diff
@@ -25,9 +25,20 @@ + (NSUserDefaults *) userDefaultsWithGroupId:(NSString * _Nullable)groupId {
 
 - (void) syncronizeToGroupId:(NSString * _Nullable)groupId {
     NSUserDefaults *destination = [NSUserDefaults userDefaultsWithGroupId:groupId];
-    [destination setObject:[self objectForKey:FWTNotifiableServerConfiguration] forKey:FWTNotifiableServerConfiguration];
-    [destination setObject:[self objectForKey:FWTUserInfoNotifiableCurrentDeviceKey] forKey:FWTUserInfoNotifiableCurrentDeviceKey];
-    [destination synchronize];
+    BOOL changed = false;
+    id serverConfiguration = [self objectForKey:FWTNotifiableServerConfiguration];
+    if (serverConfiguration) {
```

Comment:
since some compiler versions have had issues it's better/safer to compare against `nil`
```objc
if (serverConfiguration != nil) {
```